src/web-components/7-display-markers/display-markers.md

```diff
@@ -145,10 +147,11 @@ class DogElement extends HTMLElement {
 
 ✏️ Add default values for `markers` and `vehicles` in the constructor (use an underscore property for vehicles). Add a getter/setter pair for `vehicles`, where the setter creates new markers on the map with `new google.maps.Marker`.
 
+<details>
+<summary>Click to see the solution</summary>
```

Comment:
Do we have a verdict on the buses?

[Display markers for vehicles]
[ ] (Content) Is the solution supposed to show buses? Is it currently? (if not, update here & in next lessons)

Reply:
Yup, they're showing up exactly like they're supposed to. (I just didn't understand/see it when I wrote the ticket)